gio/src/dbus_node_info.rs

```diff
@@ -0,0 +1,38 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use std::ffi::CStr;
+
+use glib::translate::ToGlibPtr;
+
+use crate::{DBusInterfaceInfo, DBusNodeInfo};
+
+impl DBusNodeInfo {
+    pub fn path(self) -> &'static str {
+        let c_obj = self.to_glib_none().0;
+        let path = unsafe { (*c_obj).path };
+        if path.is_null() {
+            return "";
+        }
+        let c_str = unsafe { CStr::from_ptr(path) };
+        c_str.to_str().unwrap()
+    }
+
+    pub fn interfaces(self) -> Vec<DBusInterfaceInfo> {
```

Comment:
Do you think you can also expose the fields of the other DBus*Info types?

Reply:
I could but I currently have no use case for them. And I feel kind of bad adding too much code that is not backed up by tests.